### PR TITLE
fix: fix builds

### PR DESCRIPTION
**Pull request by Arqu**
🤷 ~~guess this fixes it?~~ or not, worked locally...

### CODE REVIEW

**Review by dignifiedquire — APPROVED** [review]
missed that one 😅

File: iroh-gateway/src/config.rs [hunk 1]
```diff
@@ -65,16 +65,18 @@ impl Config {
 
     /// Derive server addr for non memory addrs.
     pub fn server_rpc_addr(&self) -> Result<Option<GatewayServerAddr>> {
+        #![allow(clippy::unreachable)]
```
Comment (Arqu) [inline iroh-gateway/src/config.rs:68]: this doesnt help :/

Comment (dignifiedquire) [reply iroh-gateway/src/config.rs]: `#!` means apply to whole module, `#` means only next expression

Comment (dignifiedquire) [reply iroh-gateway/src/config.rs]: the warning was for `unreachable_patterns` which is a compiler lint not a clippy lint ( so no `clippy:` prefix)

Comment (Arqu) [reply iroh-gateway/src/config.rs]: 🙇

**Pull request merged**